components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit-local.jsp

```diff
@@ -502,7 +502,7 @@ function removeDefaultAuthSeq() {
                                 <font color="red">*</font>
                             </td>
                             <td>
-                                <input type="text" id="destinationURLTxt" class="text-box-big" value="" white-list-patterns="http-url https-url"/>
+                                <input type="text" id="destinationURLTxt" value="" white-list-patterns="http-url https-url"/>
```

Comment:
Removing the `text-box-big` class will not effect the style of the component ?

Reply:
No. The class text-box-big only has the style `width: 250px;` so, removing it will set the input box size to the minimum width set which is `min-width: 230px;`

Reply:
Ok. Then no prob.